src/test/resources/features/DashboardPage.feature

```diff
@@ -47,6 +47,23 @@ As a registered user
     Then I close Draft guidance page tab
     And I should navigate back to dashboard page
     And I should be on Dashboard page
+    When I click Sign out link
+    Then I am on feedback survey page
+
+
+    Scenario: 2 - Validate sign-in to dashboard page for existing entity
+      Given Organisation User logs in Dashboard page for Pillar2
+      Then I should be on Dashboard page
```

Comment:
you should logon tom task list and completed the details, this scenarios seems incorrect